apps/condo/domains/common/utils/timezone.js

```diff
@@ -13,6 +15,15 @@ const normalizeTimeZone = (timeZone) => {
     }
 }
 
+const dayjsWithTimezone = ({ date, timezone }) => {
```

Comment:
Why not just?
```
if (timeZone && normalizeTimeZone(timeZone)) {
    return dayjs(date).tz(timezone)
}

return dayjs(date)
```

Reply:
Removed this function

Reply:
Now I use the normalized or default timezone for generate blank